sample-configs/cri-resmgr-configmap.example.yaml

```diff
@@ -297,6 +297,8 @@ data:
   rdt: |+
     # Common options
     options:
+      # Mode must be one of Full, Discovery or Disabled
```

Comment:
Should we mention here that the operating mode defaults to `Full` ? Otherwise I'd suspect most people would assume it to be 'Disabled'. AFAICT it's not mentioned in the docs either. And even if it was, for those 99%'s sake who look at sample configs instead of reading the docs, I'd still put it also here...

Reply:
Very good point, fixed